tests/App.spec.js

```diff
@@ -28,4 +22,8 @@ describe('Application test', () => {
 
     expect(toJson(wrapper)).toMatchSnapshot();
   });
+  it('should render without crashing', () => {
+    const wrapper = shallow(<Menu />);
```

Comment:
Mount the component using Enzyme's `mount` method.

Also, assert that the dropdown contains expected DOM nodes e.g what links are rendered and their `href` etc.

Reply:
Okay, we will do that. Thank you for the feedback.